astropy/stats/jackknife.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from scipy.special import erfinv
```

Comment:
@crawfordsm - But scipy is only used in one of the two functions of the module, so if there is any use case for using ``jackknife_resampling`` on its own, I would argue that the import should go back to the function that uses it.